src/d3a/d3a_core/sim_results/file_export_endpoints.py

```diff
@@ -64,10 +66,9 @@ def _populate_plots_stats_for_supply_demand_curve(self, area):
                 self.cumulative_offers[area.slug][market.time_slot] = {}
                 self.cumulative_bids[area.slug][market.time_slot] = {}
                 self.clearing[area.slug][market.time_slot] = {}
-            self.cumulative_offers[area.slug][market.time_slot] = market.state.cumulative_offers
-            self.cumulative_bids[area.slug][market.time_slot] = market.state.cumulative_bids
-            self.clearing[area.slug][market.time_slot] = \
-                bid_offer_matcher.matcher.match_algorithm.state.clearing
+            self.cumulative_offers[area.slug][market.time_slot] = area.state.cumulative_offers
```

Comment:
I fail to see where this `state` is initiated in the area class. can you please help me?